### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by nokafor**
Updated the issue template per discussion

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (TCourtneyOwen) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]: Maybe something like "Create a report to help fix issues or provide feature requests"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+title: ''
+labels: needs triage
+assignees: ''
+
+---
+
+# Prerequisites
+
+Please answer the following questions for yourself before submitting an issue.
```
Comment (TCourtneyOwen) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: I think remove "for yourself"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.
+
+* Operating System:
+* Node version:
+* Office version:
+* Tool version:
+
+## Failure Logs
+
+Please include any relevant log snippets or files here.
```
Comment (TCourtneyOwen) [inline .github/ISSUE_TEMPLATE/bug_report.md:49]: Please include any relevant log snippets, screenshots or code samples here.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (nokafor) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Updated!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (nokafor) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Updated!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (nokafor) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Updated!

**Comment by nokafor** [comment]
Ok I made all the changes. @arttarawork and @TCourtneyOwen can you approve?

**Pull request merged**